netbull/security-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "NetBull\\SecurityBundle\\NetBullSecurityBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "vendor/bin/browscap-php browscap:fetch": "php-script",
```

Comment:
Could you tell me why these lines are needed?

Reply:
the idea is to use browscap for creating browser fingerprint and store which will be used to ban after X attempts and for X seconds. Browscap need to create his cache after installing and that's why the commands are needed.

Reply:
Yeah, I understand that. But I do not see that your bundle uses browscap other then in composer.json.

Reply:
Ahh, I'm still developing... after 30 min it will be ready.. :/

Reply:
now it should be fine :)